app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -67,14 +74,24 @@ export const FormAudience = ({
     register,
     reset,
     errors,
-    formState: { isValid, isSubmitted, touched },
+    getValues,
+    formState,
   } = useForm({
     mode: "onTouched",
     defaultValues,
   });
+  const { FormErrors, formControlAttrs, isValid, isSubmitted } = useCommonForm(
```

Comment:
Haven't looked too far into this PR yet, but: Any reason not to fully wrap and pass through the `useForm` hook from within `useCommonForm`?

Reply:
Ooo, that could be a good idea especially because calling `useCommonForm` takes in so many required parameters as-is from `useForm`. Will investigate😈

Reply:
Much cleaner this way!